front/lib/auth.ts

```diff
@@ -356,11 +356,12 @@ export class Authenticator {
 
     return {
       workspaceAuth: new Authenticator({
-        workspace,
-        role,
+        flags: workspaceFlags,
         groups: [],
+        key: key.toAuthJSON(),
```

Comment:
I need to refactor a few call sites so we can remove it from here. Will do in follow up PRs.